lib/seed_helper.rb

```diff
@@ -138,6 +139,9 @@ def import_raw_template(contents, vendor = 'Foreman')
     def import_templates(template_paths, vendor = 'Foreman')
       template_paths.each do |path|
         import_raw_template(File.read(path), vendor)
+      rescue RuntimeError => e
+        Foreman::Logging.exception("Error in seeding the template metadata #{e.message}", exception)
```

Comment:
what `exception` refers to? shouldn't it be `e`?

Reply:
Right, probably a copy-paste-error :innocent: 

fixed and rebased :smile: